USAGE.md

```diff
@@ -145,6 +145,40 @@ export default () => {
   );
 };
 ```
+# Web configuration
+
+## Metro bundler
+No additional steps are required when using Metro bundler.
+
+## Webpack
+If you are using the Webpack bundler, the following steps are needed:
+
+- Install `@react-native/assets-registry/registry` as a project dependency.
+- Ensure that you have the proper configuration to parse flow files from `@react-native/assets-registry/registry`.
+- Configure the Webpack `module -> rules` section and include an important rule for `node_modules/@react-native/assets-registry/registry`.
+
+#### Webpack Configuration
```

Comment:
Useless header, you can add the filename instead.
It's `babel.config.js` right? Then simply
```suggestion
babel.config.js
```

Reply:
it's a webpack.config.js, I changed. that, thank you